app/controllers/admin/druid_search_controller.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Admin
+  # Controller for searching for work or collection by druid
+  class DruidSearchController < Admin::ApplicationController
+    def new
+      authorize!
```

Comment:
I assume this controller is automagically using this policy: https://github.com/sul-dlss/hungry-hungry-hippo/blob/main/app/policies/admin_policy.rb Confirm/deny?

Reply:
It's not magic, it's code: https://github.com/sul-dlss/hungry-hungry-hippo/blob/main/app/controllers/admin/application_controller.rb

Reply:
Sweet! TIL, thanks